### DISCUSSION TITLE
GraphQL - filter based on linked collection's data

### DISCUSSION THREAD

**Discussion by nithinkashyapn** [statamic/cms#4862] [Q&A]
Hey,

Using the GraphQL API is there a way to filter based on linked collection's data?

Something like -

```
query MyQuery {
  entries(collection: "jobs", filter: {company: {title: {contains: "Acme"}}}) {
    data {
      ... on Entry_Jobs_Jobs {
        id
        company {
          title
        }
        title
      }
    }
  }
}

```

Where company is another collection linked using the [Entries](https://statamic.dev/fieldtypes/entries) field type

**Comment by christie-hill-za** [comment]
I sincerely apologise for the horrible necro, but I have the same question.

**Reply by lotarbo** [reply]
yep, i want the same. Filtering base on related entry slug)
